pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -938,6 +942,11 @@ func (cr *PerconaXtraDBCluster) SetVersion() bool {
 	return true
 }
 
+func (cr *PerconaXtraDBCluster) ValidateVersion() error {
```

Comment:
Can this method be private? I don't see it is used anywhere else except in this package.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1192/commits/b6eee1a3034b5cb343d852814e9f82330af44218